README.md

```diff
@@ -311,6 +311,27 @@ Plot automatically generates axes for position scales. You can configure these a
 
 Top-level options are also supported as shorthand: **grid** (for *x* and *y* only; see [facet.grid](#facet-options)), **label**, **axis**, **inset**, **round**, **align**, and **padding**.
 
+### Projection options
+
+The top-level **projection** option applies a (often geographic) projection in place of *x* and *y* scales. It is typically used in conjunction with [geometry marks](#geometry) to produce a map, but can be used with any mark that supports *x* and *y*, such as [dot](#dot) and [text](#text). The following built-in named projections are supported:
+
+* *equirectangular* - the equirectangular, or *plate carrée*, projection
+* *orthographic* - the orthographic projection
+* *stereographic* - the stereographic projection
+* *mercator* - the Mercator projection
+* *equal-earth* - the [Equal Earth projection](https://en.wikipedia.org/wiki/Equal_Earth_projection) by Šavrič *et al.*
+* *natural-earth* - the [Natural Earth projection](https://en.wikipedia.org/wiki/Natural_Earth_projection) by Patterson
+* *albers-usa* - a composite Albers conic equal-area projection suitable for the United States
```

Comment:
```suggestion
* *natural-earth* - the [Natural Earth projection](https://en.wikipedia.org/wiki/Natural_Earth_projection) by Patterson
* *azimuthal-equal-area* - the azimuthal equal-area projection
* *azimuthal-equidistant* - the azimuthal equidistant projection
* *conic-conformal* - the conic conformal projection
* *conic-equal-area* - the conic equal-area projection
* *conic-equidistant* - the conic equidistant projection
* *gnomonic* - the gnomonic projection
* *transverse-mercator* - the transverse Mercator projection
* *albers* - the Albers’ equal-area conic projection
* *albers-usa* - a composite Albers conic equal-area projection suitable for the United States
```

TBH I'm not sure we should support (or document) all these projections in v1; for example, the generic conic projections are pretty useless unless we can parametrize the parallels.

Reply:
You can specify parallels like so:

```js
projection: {
  type: "conic-equal-area",
  parallels: [30, 60]
}
```

The supported options are shown here:

https://github.com/observablehq/plot/blob/4b561a4080e170254e13dbd92d616d3db8d7ddbf/src/projection.js#L95-L98